cmd/daemon/main.go

```diff
@@ -487,10 +487,16 @@ func main() {
 		grpc.Creds(internal.NewUnixSocketCredentials(internal.NewDaemonAuthenticator())),
 	}
 
-	norduserMonitor := norduser.NewNordVPNGroupMonitor(norduserService)
+	norduserMonitor := norduser.NewNorduserProcessMonitor(norduserService)
 	go func() {
-		if err := norduserMonitor.Start(); err != nil {
-			log.Println("Error when starting norduser monitor: ", err.Error())
+		if snapconf.IsUnderSnap() {
+			if err := norduserMonitor.StartSnap(); err != nil {
+				log.Println("Error when starting norduser monitor for snap: ", err.Error())
```

Comment:
I would suggest to add `internal.XYZPrefix` to the log. Maybe `internal.WarningPrefix` here since it's not changing the control flow?
Also, you don't need space after ":" when you are using `log.Println`, it will be added by `Println`